### DISCUSSION TITLE
about loaderUrl, dataUrl, use cdn?

### DISCUSSION THREAD

**Discussion by qld-cf** [jeffreylanters/react-unity-webgl#431] [Question and Answer]
/**
     * The url to the build json file generated by Unity. When using a relative url,
     * keep in mind this is relative from the path where your html file is served.
     */

like unity-config.d.ts description, loaderUrl and so on, just can put these files to local project?
can we use cdn file url?
@jeffreylanters

**Comment by jeffreylanters** [comment]
Hi! Yes, that's also possible! I'll update the descriptions to reflect that.
